src/main/java/spoon/support/reflect/CtExtendedModifier.java

```diff
@@ -25,15 +25,48 @@
 	private ModifierKind kind;
 	private SourcePosition position;
 
+	/**
+	 * Creates a new extended modifier of the given kind.
+	 *
+	 * @param kind the kind of this modifier.
+	 * @deprecated use {@link #explicit(ModifierKind)} to create an explicit modifier.
+	 */
+	@Deprecated
```

Comment:
Maybe add a `since` value like 9.2? [https://download.java.net/java/GA/jdk14/docs/api/java.base/java/lang/Deprecated.html#since()](https://download.java.net/java/GA/jdk14/docs/api/java.base/java/lang/Deprecated.html#since())